quiz.js

```diff
@@ -17,9 +17,22 @@ db.dropDatabase();
 
 // your code begins here
 
+use houses
```

Comment:
Not quite. `use` is used to select a database (like `westeros`), not a collection (like `houses`), and we've already said in line 6 that our database is `westeros`. Moreover, it's not necessary to select a collection - you can simply start using it (once you're on the appropriate database).